dockers/docker-orchagent/Dockerfile.j2

```diff
@@ -28,14 +28,14 @@ RUN apt-get update &&       \
 # Fix for gcc/python/iputils-ping not found in arm docker
 RUN apt-get install -f -y python2.7 python2.7-dev
 RUN apt-get install -y   \
-        gcc-8            \
+        gcc-10           \
         iputils-ping
 {% endif %}
 {% if CONFIGURED_ARCH == "armhf" %}
-RUN ln -s -f /usr/bin/gcc-8 /usr/bin/arm-linux-gnueabihf-gcc
+RUN ln -s -f /usr/bin/gcc-10 /usr/bin/arm-linux-gnueabihf-gcc
```

Comment:
I think if the above is changed to `gcc` instead of `gcc-10`, then these symlinks won't be needed.